src/internal/wasm.ts

```diff
@@ -45,7 +45,7 @@ export const instantiateWasm = async (
   internal.wasmModules.set(wasmId, instantiated);
 };
 
-export const readResponse = (pointer: bigint, buffer: Uint8Array): any => {
```

Comment:
I basically punted here as this export that was imported in the same file was causing errors in generated code in commonjs